examples/formats/NEXRAD_Level_2_File.py

```diff
@@ -47,23 +49,33 @@
 rho_range = (np.arange(rho_hdr.num_gates + 1) - 0.5) * rho_hdr.gate_width + rho_hdr.first_gate
 rho = np.array([ray[4][b'RHO'][1] for ray in f.sweeps[sweep]])
 
+# Extract central longitude and latitude from file
+cent_lon = f.sweeps[0][0][1].lon
+cent_lat = f.sweeps[0][0][1].lat
 ###########################################
-fig, axes = plt.subplots(1, 2, figsize=(15, 8))
+gs = gridspec.GridSpec(1, 2)
+fig = plt.figure(figsize=(15, 8))
 add_metpy_logo(fig, 190, 85, size='large')
-for var_data, var_range, ax in zip((ref, rho), (ref_range, rho_range), axes):
+
+for var_data, var_range, ax in zip((ref, rho), (ref_range, rho_range), gs):
```

Comment:
Would be good to use something like `ax_rect` rather than `ax` here since it's a bit confusing that `ax` isn't actually a matplotlib `Axes` class.